pkg/operator/controller/status.go

```diff
@@ -51,20 +58,18 @@ func (r *reconciler) syncOperatorStatus() error {
 		},
 	}
 
-	if len(r.OperatorReleaseVersion) > 0 {
```

Comment:
Removed ^ since `r.OperatorReleaseVersion` is now set to "unknown" if env var is not set.